pkg/query/query.go

```diff
@@ -401,3 +407,87 @@ func runContinuousQueries(client *http.Client, requestStr string) {
 		_ = sendSingleRequest(freeText, client, fQuery, requestStr, true)
 	}
 }
+
+// Run queries from a csv file
+
+func RunQueryFromFile(dest string, numIterations int, prefix string, continuous, verbose bool, filepath string) {
+	// open file
+	f, err := os.Open("queries.csv")
+	if err != nil {
+		log.Infof("RunQueryFromFile: Error in opening file: %v, err: %v", filepath, err)
+	}
+
+	defer f.Close()
+
+	// read csv values using csv.Reader
+	csvReader := csv.NewReader(f)
+	for {
+		rec, err := csvReader.Read()
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			log.Infof("RunQueryFromFile: Error in reading file: %v, err: %v", filepath, err)
+		}
+		// do something with read line
+		fmt.Printf("%+v\n", rec)
```

Comment:
use the already imported `log` module & maybe we can put such statements behind the `verbose` flag